src/trimgalore/config.vsh.yaml

```diff
@@ -0,0 +1,314 @@
+name: trimgalore
+description: | 
+  A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files. 
+keywords: ["trimming", "adapters"]
+links:
+  homepage: https://github.com/FelixKrueger/TrimGalore
+  documentation: https://github.com/FelixKrueger/TrimGalore/blob/master/Docs/Trim_Galore_User_Guide.md
+  repository: https://github.com/FelixKrueger/TrimGalore
+references: 
+  doi: 10.5281/zenodo.7598955
+license: GPL-3.0 
+requirements:
+  commands: [trim_galore]
+authors:
+  - __merge__: /src/_authors/sai_nirmayi_yasa.yaml
+    roles: [ author, maintainer ]
+
+argument_groups:
+  - name: Input
+    arguments:   
+    - name: "--input"
+      type: file
+      description: Input files. Note that paired-end files need to be supplied in a pairwise fashion, e.g. file1_1.fq file1_2.fq SRR2_1.fq.gz SRR2_2.fq.gz
+      required: true
+      multiple: true
+      example: sample1_r1.fq;sample1_r2.fq;sample2_r1.fq;sample2_r2.fq
+  - name: Trimming options
+    arguments: 
+      - name: --quality
+        alternatives: -q
+        type: integer
+        description: Trim low-quality ends (below the specified Phred score) from reads in addition to adapter removal. For RRBS samples, quality trimming will be performed first, and adapter trimming is carried in a second round. Other files are quality and adapter trimmed in a single pass. The algorithm is the same as the one used by BWA (Subtract INT from all qualities; compute partial sums from all indices to the end of the sequence; cut sequence at the index at which the sum is minimal). 
+        example: 20
+        required: false
+      - name: --phred33
+        type: boolean
+        description: Instructs Cutadapt to use ASCII+33 quality scores as Phred scores (Sanger/Illumina 1.9+ encoding) for quality trimming. 
+        required: false
+      - name: --phred64
+        type: boolean
+        description: Instructs Cutadapt to use ASCII+64 quality scores as Phred scores (Illumina 1.5 encoding) for quality trimming.
+        required: false
+      - name: --fastqc
+        type: boolean
+        description: Run FastQC in the default mode on the FastQ file once trimming is complete.
+        required: false
+      - name: --fastqc_args
+        type: string
+        description: Passes extra arguments to FastQC. If more than one argument is to be passed to FastQC they must be in the form "arg1 arg2 ...". Passing extra arguments will automatically invoke FastQC, so --fastqc does not have to be specified separately.
+        required: false
+        example: "--nogroup --outdir /home/"
+      - name: --adapter
```

Comment:
Please revisit: IMO it's better to turn this into 2 arguments: one for strings, one for files?

Reply:
In this case, I'm not sure if `type: file` would work since both the argument name and value should be specified. For example, if we want to pass adapters and contaminants as additional inputs to fastqc we would do something like `fastqc_args "-a adapter_list.txt -c contaminants_list.txt"`. This would be hard to handle with file type argument.

Reply:
I see. Do you have a test case for this? I'm quite sure those two files in your example will not be considered as input by Viash and later Nextflow and so no pointers to those files will exist when used in a workflow.

An alternative may be to do something like this:

```yaml
  - name: fastqc_args
    type: string
    example: "-a adapter_list.txt -c contaminants_list.txt"
  - name: fastqc_args_files
    type: file
    multiple: true
    example: [ adapter_list.txt, contaminants_list.txt ]
  ```
  
  The description should make clear then what is the link between both.
  
  wdyt?

Reply:
Since there weren't many files arguments, I changed it such that each of the files can be specified separately (`--fastqc_contaminants`, `--fastqc_adapters`, `--fastqc_limits`) and all the others are specified with `--fastqc_args`.